.README/rules/no-undefined-types.md

```diff
@@ -47,11 +47,18 @@ array's items will be considered as defined for the purposes of that tag.
 
 ## Options
 
-An option object may have the following key:
+An option object may have the following keys:
 
 - `definedTypes` - This array can be populated to indicate other types which
   are automatically considered as defined (in addition to globals, etc.).
   Defaults to an empty array.
+- `markVariablesAsUsed` - Whether to mark variables as used for the purposes
+  of the `no-unused-vars` rule when they are not found to be undefined.
+  Defaults to `true`. May be set to `false` to enforce a practice of not
+  importing types unless used in code.
+- `disableReporting` - Whether to disable reporting of errors. Defaults to
+  `false`. This may be set to `true` in order to take advantage of marking
```

Comment:
I suppose this is exposed for developer that just want the marking af of types as used.

```suggestion
  `false`. This may be set to `true` in order to take advantage of only marking
```